scripts/maestro

```diff
@@ -672,7 +672,11 @@ def add_updaters(comms, updaters, aggregators, daemons, agg_state, rb=None):
                     print(f'Error {err}: getting {agg_name} current updater status')
                     print(f'{agg_name} has become unresponsive since last rebalance. Rebalancing...')
                     return 107
-                prdcr_updtr_status = json.loads(rc)
+                try:
+                    prdcr_updtr_status = json.loads(rc)
+                except Exception as e:
+                    print(f'Error: {e} parsing updtr status from {agg_name}')
```

Comment:
Does the json plugin provide more detailed information regarding the location of the syntax error in the file. I think this is just the generic error, but maybe not.